### PR TITLE
policystream - fix the identification of removed policies

### PR DESCRIPTION
**Pull request by howbazaar**
The set difference was the wrong way around for identifying which policies
to identify in the previous set. The extra logic for only trying to identify
the policies that were not moved is not necessary, as evident by the added
test.

### CODE REVIEW

File: tools/c7n_policystream/policystream.py [hunk 1]
```diff
@@ -399,11 +399,7 @@ def _process_stream_commit(self, change):
                     current_policies += self.policy_files[f]
             elif delta.status == GIT_DELTA_INVERT['GIT_DELTA_DELETED']:
                 if f in self.policy_files:
-                    # if the policies were moved, only add in policies
```
Comment (kapilt) [inline tools/c7n_policystream/policystream.py:402]: I think the intent was for policies moved in the same commit, where as the tests added here model remove and add in separate commits

Comment (howbazaar) [reply tools/c7n_policystream/policystream.py:402]: The second test moves a policy from the removed file to another file.

What do you mean by "moved in the same commit" if it isn't the second test case?

File: tools/c7n_policystream/tests/test_policystream.py [hunk 2]
```diff
@@ -187,3 +192,50 @@ def test_cli_stream_basic(self):
...
+             ('remove', 'lambda-check', 'remove file')])
+
+    def test_stream_move_policy(self):
+        git = self.setup_basic_repo()
+        git.change('newfile.yml', {
+            'policies': [{
+                'name': 'ec2-check',
+                'resource': 'aws.ec2'}]})
+        git.commit('new file')
+        git.rm('example.yml')
```
Comment (kapilt) [inline tools/c7n_policystream/tests/test_policystream.py:220]: per other thread on a test case, deleting a file is a different consideration then simply moving a policy. there isn't a policy file to delta against in that case afaics which is what the logic being removed was doing.

Comment (kapilt) [reply tools/c7n_policystream/tests/test_policystream.py:220]: i'll play around with it.

File: tools/c7n_policystream/policystream.py (hunk 1 above)
Comment (kapilt) [reply tools/c7n_policystream/policystream.py:402]: there's a couple cases for move, a simple file rename, or a content change move. i added an additional test case for the rename, but the implementation change looks good.

**Review by kapilt — APPROVED** [review]
thanks, lgtm

**Pull request merged**